### PR TITLE
added train_model.py and main.py, data folder with raw data and processed folder, added code on makedataset to look at len of dim 3

### PR DESCRIPTION
**Pull request by Jddumas**

### CODE REVIEW

File: src/models/train_model.py [hunk 1]
```diff
@@ -54,43 +48,29 @@ def forward(self, x):
         return F.softmax(x, dim=1)
 
 class Model():
-    def __init__(self):
-        """Constuctor method for TrainModel class can tweak epoch and batch_size"""
+    def __init__(self, batch_size, epochs, learning_rate, validation_percent, training_data):
```
Comment (allisonglittle) [inline src/models/train_model.py:51]: Nice call adding these, I think this will help us run different versions of the model.

**Review by allisonglittle — APPROVED** [review]
I understand that you feel the need to include the .wav files in this commit for the Project Midpoint review, but I maintain that we will want to avoid doing this with our entire dataset going forward.
Nice job, Jacob. This model training code looks really good and I appreciated the demo yesterday.

**Review by mideenraj — APPROVED** [review]
Great work on creating the training model!
